lib/ui/viewer/file/file_info_dialog.dart

```diff
@@ -437,4 +326,167 @@ class _FileInfoWidgetState extends State<FileInfoWidget> {
       },
     );
   }
+
+  Widget _getVideoDuration() {
+    if (widget.file.duration != 0) {
+      return Text(
+        secondsToHHMMSS(widget.file.duration),
+      );
+    }
+    return FutureBuilder(
+      future: widget.file.getAsset(),
```

Comment:
file.getAsset() will not work when
- File is not present locally
- File is shared to ente app.

Also, we will need to hide the video information for live photos. 
We can may be add this check for now..
Call `_getVideoDuration` only when 
`(file.type == Video and (file.localID != null or file.duration !=0)`
And replace
`widget.file.getAsset()`
with `getFile()` (present in file_util)

Reply:
For local files file.duration = 0. So file.getAsset() is only called for local files (see if condition on line 331). I think that is also the case for videos shared to ente app because it was working fine in the app when I ran it (the duration for shared videos). 

Yes, will hide video information for for live photos.

Reply:
Sorry, we don't need to call getFile. Only need to call file.getAsset() for a local file, which not shared to the app.